soc/arm/nuvoton_npcx/Kconfig

```diff
@@ -20,6 +20,21 @@ menuconfig NPCX_HEADER
 
 if NPCX_HEADER
 
+config NPCX_IMAGE_OUTPUT_BIN
+	bool "Build npcx binary in BIN format"
+	default y
+	help
+	  Build a "raw" binary zephyr/zephyr.npcx.bin in the build directory.
+	  The name of this file can be customized with CONFIG_KERNEL_BIN_NAME.
+
+config NPCX_IMAGE_OUTPUT_HEX
```

Comment:
why don't you just use the generic option BUILD_OUTPUT_HEX defined in Kconfig.zephyr?

Reply:
Please check the previous one.